airbyte-webapp/src/locales/en.json

```diff
@@ -485,6 +485,8 @@
   "tables.connectionDeleteConfirm": "Confirm connection deletion",
   "tables.sourceDeleteModalText": "Deleting a source cannot be undone without a full re-sync. Note that:\n  - All connections with this source will be deleted, including their past logs and configurations.\n  - No existing data in the destination will be altered",
   "tables.destinationDeleteModalText": "Deleting a destination cannot be undone. Note that:\n  - All connections with this destination will be deleted, including their past logs and configurations.\n  - No existing data in the destination will be altered.",
+  "tables.affectedConnectionsOnSourceDeletion": "This connections will be affected if you delete this source:\n",
+  "tables.affectedConnectionsOnDestinationDeletion": "This connections will be affected if you delete this destination:\n",
```

Comment:
It should be "The following connection will be deleted" instead of "This connections will be affected"

Reply:
Do we have a library for pluralizing? Ideally it would be
1 connection: "The following connection..."
2+ connections: "The following connections..."

Reply:
Yes, the i18n library can pluralize